tests/test/dashboard.test.js

```diff
@@ -74,4 +78,45 @@ describe('The Unlock Dashboard', () => {
       await expect(page).toMatch('Updated Lock Name')
     })
   })
+
+  // TODO: figure out why these will not pass in a separate file
```

Comment:
Since these tests are not for the dashboard I think we should really actually them in a separate file first.
Could it be that for some reason we *need* to do to `/dashboard` first to initialize the app completely?